### PR TITLE
Export other ActiveStorage JavaScript modules.

### PR DESCRIPTION
**Pull request by fnando**
### Summary

With this change, devs that need to make custom direct upload scripts can import the module directly from `@rails/activestorage` instead of `@rails/activestorage/src/direct_upload_controller`.

### CODE REVIEW

**Comment by rafaelfranca** [comment]
Can you change the target branch? Since this is more like a "feature" than a bug we can't apply anymore on 7.0.x

**Comment by fnando** [comment]
@rafaelfranca done!

File: activestorage/app/javascript/activestorage/index.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { start } from "./ujs"
 import { DirectUpload } from "./direct_upload"
-export { start, DirectUpload }
+import { DirectUploadController } from "./direct_upload_controller"
```
[the remark this answers is not in the document]
Comment (fnando) [reply activestorage/app/javascript/activestorage/index.js:3]: `DirectUploadController` (singular) is actually the module I'm using to handle drag 'n' drop file uploading with direct uploading. I don't think `DirectUploadsController` (plural) allows for that use case, as it doesn't take a file instance, as opposed to the singular-named class.

The gist of the code would be

```js
const controller = new DirectUploadController(
  this.fileInputTarget,
  this.file,
);

controller.start((error: Error) => {
  console.log("ended", { error });

  if (!error) {
    (this.element as HTMLFormElement).submit();
  }
});
```

**Comment by rafaelfranca** [comment]
Looks good me to. I think you need to update the compiled file with `cd activestorage; rollup --config rollup.config.js` and commit it.

**Comment by fnando** [comment]
@rafaelfranca done!

**Pull request merged**